libmalcontent/tests/app-filter.c

```diff
@@ -636,6 +651,21 @@ get_app_filter_server_cb (GtDBusQueue *queue,
   properties_variant = g_variant_ref_sink (g_variant_new_parsed (data->properties));
   g_dbus_method_invocation_return_value (invocation2,
                                          g_variant_new_tuple (&properties_variant, 1));
+
+  /* Handle the Properties.Get() call for the AccountType, and say the account
+   * is a normal user. */
```

Comment:
Not a new issue with the rebase but this comment isn't quite right. You are returning `data->account_type`, not always saying it's a normal user.

Reply:
Squashed a fix for that into our downstream commit, thanks.